docs/howto/configure/auth-management.md

```diff
@@ -166,3 +166,59 @@ Presently, this involves a few more manual steps than the `auth0` setup describe
     ```
 
 4. Run the deployer as normal to apply the config.
+
+# CILogon
+
+For communities that require authenticating users against their campus identity providers, [CILogon](https://www.cilogon.org) can be used. CILogon authentication is managed through [auth0](https://auth0.com), similar with how the Google and GitHub authentication is setup.
+
+```{note}
+  The `USERNAME_KEY` used for the CILogon login is the email, so the JupyterHub username will be set to the email address associated with the CILogon connection and not the CILogon user_id!
+```
+
+To enable CILogon authentication:
+
+1. Explicitly list CILogon as the type of connection we want for a hub, via `auth0.connection`:
+
+   ```yaml
+   auth0:
+      connection: CILogon
+   ```
+
+2. Explicitly list *admin users* for a given hub and only allow the users that match specific identity providers to login into the hub.
+
+  ```{note}
+  Don't forget to allow login to the test user (`deployment-service-check`), otherwise the hub health check performed during deployment will fail.
```

Comment:
Isn't this something that we can automatically add at deploy time? It isn't configuration a community would ever want to copy over themselves if they deployed their own hub

Reply:
Yeah, so if I remember correctly from when I struggled to have a nice way to add the staff by default to the admin&allowed lists, I don't think there's a straightforward way of doing this. And I don't think we should stress out about this as this `username_pattern` thing it's already pretty ugly as it is.

We should be able to get away from this if we manage to automatize the CILogon client creation and move out from Auth0 entirely.